### ISSUE TITLE
Image-only Grid list of material ui next

### ISSUE THREAD

**Issue by knsakib** [xotahal/react-native-material-ui#339]
Did you implement this 
https://material-ui-next.com/demos/grid-list/#grid-list
in your react-native-material-ui version?

**Comment by ayushyamitabh** [comment]
No, this doesn't exist yet. If you want to help and create it I'm sure @xotahal would accept it.

**Comment by peacechen** [comment]
The grid is rather generic and not necessarily specific to material design.  There exists a good RN library for responsive girds.  It could be incorporated into this library, or used separately by the app as needed.

https://github.com/idibidiart/react-native-responsive-grid
